client/common/cmdline.c

```diff
@@ -1465,15 +1465,19 @@ int freerdp_client_settings_command_line_status_print_ex(rdpSettings* settings,
 			settings->ListMonitors = TRUE;
 		}
 
-		return COMMAND_LINE_STATUS_PRINT;
+		goto out;
 	}
 	else if (status < 0)
 	{
 		freerdp_client_print_command_line_help_ex(argc, argv, custom);
-		return COMMAND_LINE_STATUS_PRINT_HELP;
+		status = COMMAND_LINE_STATUS_PRINT_HELP;
```

Comment:
This is not right as it consequently returns 0 also for command-line parsing errors. It should return 0 only if `status == COMMAND_LINE_STATUS_PRINT_HELP`, shouldn't it?

Reply:
this is exactly the same as it was before, but will check.